test/relayclient/RelaySelectionManager.test.ts

```diff
@@ -57,8 +66,15 @@ contract('RelaySelectionManager', function (accounts) {
     let stubGetNextSlice: SinonStub
 
     before(async function () {
+      contractInteractor = await new ContractInteractor({
+        provider: web3.currentProvider as HttpProvider,
+        logger
+      }).init()
+      httpClient = new HttpClient(new HttpWrapper(), this.logger)
```

Comment:
different logger? why "this" ?

Reply:
A mistake. It did not crash anything because the client's methods are immediately stubbed.